ARTICLE-2.markdown

```diff
@@ -2,8 +2,8 @@
 
 ### Section 2.1  Voluntary Withdrawal
 
-A Member shall have the right to resign from the Corporation, effective
-immediately without any action on the part of the Corporation.
+A Member shall have the right to resign from the Cooperative, effective
+immediately without any action on the part of the Cooperative.
 
```

Comment:
Well, lets see what members say. I have a feeling all will agree to this. We do not have cash flow to refund @brodavi, its a practical thing. Can we add it please?